rules/no-focused-tests.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
   create: context => ({
     CallExpression(node) {
       const callee = node.callee;
+      /* istanbul ignore if */
       if (!callee) {
```

Comment:
@jkimbo this is an old one, can you provide details on the cases where a `CallExpression` has no `callee`?